src/server/auth/cmds/cmds.go

```diff
@@ -16,6 +19,18 @@ import (
 
 var githubAuthLink = `https://github.com/login/oauth/authorize?client_id=d3481e92b4f09ea74ff8&redirect_uri=https%3A%2F%2Fpachyderm.io%2Flogin-hook%2Fdisplay-token.html`
 
+// StripGRPCCode removes GRPC error code information from 'err' if it came from
+// GRPC (and returns it unchanged otherwise)
+func StripGRPCCode(err error) error {
+	if err == nil {
+		return nil
+	}
+	if s, ok := status.FromError(err); err != nil && ok {
+		return errors.New(s.Message())
+	}
+	return err
+}
+
```

Comment:
I think we already have a method called `sanitizeErr` that we've been using to do effectively this same thing. Maybe this is a good opportunity the throw this method in a common place and use it throughout the code base,  I think we copy paste the method around to a few places.

Reply:
This is fixed by https://github.com/pachyderm/pachyderm/pull/2289